src/widgets/attachments/attachments-overview.component.tsx

```diff
@@ -65,10 +67,52 @@ export default function AttachmentsOverview() {
     }
   }
 
+  function handleDragOver(e: React.SyntheticEvent) {
+    e.preventDefault();
+    e.stopPropagation();
+  }
+
   function handleCurrentImageChange(index) {
     setCurrentImage(index);
   }
 
+  function handleImageSelect(index, image) {
+    const attachments_tmp = attachments.slice();
+    const attachment = attachments_tmp[index];
+    if (attachment.hasOwnProperty("isSelected")) {
+      attachment.isSelected = !attachment.isSelected;
+    } else {
+      attachment.isSelected = true;
+    }
+    setAttachments(attachments_tmp);
+  }
+
+  function getSelectedImages() {
+    let selected = [];
+    attachments.forEach((att, index) => {
+      if (att.isSelected === true) {
+        selected.push(att);
+      }
+    });
+    return selected;
+  }
+
+  function deleteSelected() {
+    const selected = attachments.filter(att => att.isSelected === true);
+    const abortController = new AbortController();
+    const result = Promise.all(
+      selected.map(att =>
+        deleteAttachment(att.id, abortController).then((response: any) => {})
+      )
+    );
+    result.then(() => {
+      const attachments_tmp = attachments.filter(
+        att => att.isSelected !== true
+      );
+      setAttachments(attachments_tmp);
```

Comment:
Dangerous - if we route to another component while this is in progress we'll get an error.

We need to cancel the promise (or not run the `setAttachments`) when the component unmounts.